testing/terraform/ec2/main.tf

```diff
@@ -66,7 +66,7 @@ resource "aws_instance" "main_service_instance" {
   ami                                   = data.aws_ami.ami.id # Amazon Linux 2 (free tier)
   instance_type                         = "t2.micro"
   key_name                              = local.ssh_key_name
-  iam_instance_profile                  = "APP_SIGNALS_EC2_TEST_ROLE"
+  iam_instance_profile                  = var.test_role
```

Comment:
Will this immediately break release testing in CWA and CWA Operator?

Reply:
CWA Operator doesn't use EC2, but it will break CWA